source/developers-guide/shopware-5-search-bundle/index.md

```diff
@@ -231,3 +231,85 @@ In most cases, only a list of product numbers is not enough for the store front.
 The `Shopware\Bundle\SearchBundle\ProductSearch` class is a decorator, which allows the execution of a product number search, whose result sets contains a list of `Shopware\Bundle\StoreFrontBundle\Struct\ListProduct` structs.
 A `ListProduct` struct contains all required data for a product to be display in the store front listings.
 Internally, this decorator first executes the `ProductNumberSearch` and then converts the result using the configured `Shopware\Bundle\StoreFrontBundle\Service\ListProductServiceInterface` from the DI container.
+
+## Shopware 5.3 changes
+With Shopware 5.3, a new interface implemented to handle facets: `\Shopware\Bundle\SearchBundleDBAL\PartialFacetHandlerInterface`. Each facet handler had to revert the provided criteria by their own to remove customer conditions. This behaviour is now handled in the `\Shopware\Bundle\SearchBundleDBAL\ProductNumberSearch::createFacets`
```

Comment:
"With Shopware 5.3, a new interface **is**  implemented to handle facets", or better imho: "Shopware 5.3 implements a new interface to handle facets"